org/pr/release-notes.ts

```diff
@@ -0,0 +1,37 @@
+import {warn, danger} from "danger";
+
+export default async () => {
```

Comment:
Maybe include a quick general description about what this Peril rule is about at the top of the file?
```suggestion

// This Peril rule will warn if the `RELEASE-NOTES.txt` file has been changed in a pr targeting a `release/*` branch.
//
// This file should typically not be modified after code freeze (i.e. on the release branch) because the release notes
// have already been sent to Editorial then up for translation at this point so due to our process about this,
// any changes to the file won't be automatically included in the corresponding release in practice.

export default async () => {
```

Reply:
Right! I pushed a shorter description which outlines what the rule does, but doesn't go into the process. I think that adding notes about process becomes "documentation about the process" that then needs to be maintained 😅
I prefer to decouple stuff: we have a rule that does something, and we integrate it where we need it.